vignettes/coast.Rmd

```diff
@@ -109,6 +109,17 @@ AllelicSeries::COAST(
 )
 ```
 
+* `min_mac` is used to filter the variant set to those containing a minimum minor allele count (MAC). The following example filters to only those variants with a MAC of at least 10:
+```{r, eval = FALSE}
+AllelicSeries::COAST(
+  anno = anno,
+  geno = geno,
+  pheno = pheno,
+  covar = covar,
+  min_mac = 2
```

Comment:
10 to match comment?

Reply:
Using 10 here is too strict, in filters out all variants!